domains/auth/Auth.API/API/Controllers/InvalidateController.cs

```diff
@@ -0,0 +1,54 @@
+using API.Models;
+using Microsoft.AspNetCore.Mvc;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using API.Helpers;
+using API.Services.OidcProviders;
+using FluentValidation;
+
+namespace API.Controllers;
+
+[ApiController]
+public class InvalidateController : ControllerBase
+{
+    readonly IOidcService _oidcService;
+    private readonly IValidator<AuthState?> _validator;
+    private readonly ICryptography _cryptography;
+
+    public InvalidateController(IOidcService oidcService, ICryptography cryptography,
+        IValidator<AuthState> validator)
+    {
+        _oidcService = oidcService;
+        _validator = validator;
+        _cryptography = cryptography;
+    }
+
+    [HttpPost]
+    [Route("/invalidate")]
+    public IActionResult Invalidate([FromQuery] string state)
```

Comment:
What about using [FromServices] for injection of dependencies: https://docs.microsoft.com/en-us/aspnet/core/mvc/controllers/dependency-injection?view=aspnetcore-6.0#action-injection-with-fromservices

Reply:
I will rather have this in another PR. 
Right now the controller max has 3 dependencies, but i  think it will be a great solution if we have to handle more dependencies so we can overvoid an overloaded constructor.

Will that be alright for you?

Reply:
That's fine